js/base/Exchange.js

```diff
@@ -962,7 +962,10 @@ module.exports = class Exchange {
         return balance;
     }
 
-    safeOrder (order, market = undefined) {
+    safeOrder (order, market = undefined, parseTrade = undefined) {
```

Comment:
@pcriadoperez these changes are not working in PHP

Reply:
fixed :)